lib/DxilPIXPasses/DxilDebugInstrumentation.cpp

```diff
@@ -1151,13 +1150,23 @@ DxilDebugInstrumentation::addStepDebugEntryValue(BuilderContext *BC,
                                  ValueOrdinalIndex);
     return DebugShaderModifierRecordTypeDXILStepFloat;
   case Type::TypeID::IntegerTyID:
+    if (V->getType()->getIntegerBitWidth() > 64) {
+      // If this actually happens, we'd like to catch it in debug
+      assert(false);
```

Comment:
`assert(false)` is an anti-pattern, since it's not locally informative.  Could an assert be written before these if conditions that checks the size against the set of expected sizes instead?  Something like:
```cpp
assert(V->getType()->getIntegerBitWidth() == 32 ||
       V->getType()->getIntegerBitWidth() == 64);
```
Using DXASSERT with a message would also seem helpful, but I'm not sure if that's fallen out of favor for use in passes.

Reply:
Right you are. Laziness on my part.